.bit/responses/3.1-Week 3 Step 1.md

```diff
@@ -1,76 +1,58 @@
-
-Here's a list of HTML items you need to create (please use the id's specified)
-
-1. `header` element that says anything you want. For example, mine says **Example Project**
-2. `div` element with id `container` that will surround **all** of your elements.
-    1. `form` element with id `image-form`, with `onsubmit="handle(event)"`. Set the `enctype` attribute to `multipart/form-data`. Remember that for forms that receive file uploads, we need to specify this type of encoding.
-        * The following three elements are contained within the form element:
-          * `input` element that allows a file upload, where the user will upload an image. 
-            * [This link](https://www.w3schools.com/html/html_form_input_types.asp) could be helpful.
-            * Set the `onchange` attribute to `"loadFile(event)"`. Use the `accept` attribute to only allow image submissions. Finally, set the `name` attribute to `image`.
-          * `img` element with id `output`. This is going to display the image that the user selects.
-          * `button` element with the `type` attribute set to `submit`. The text inside should say "Submit Picture" or something similar. This will submit the image.
-    2. Empty `div` with the id `emotion`. This is where the emotion analysis results will be displayed.
-
-Lastly, make sure to reference jQuery:
-
-```html
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
-     
-    <script src="config.js" type="text/javascript"></script>
-```
-
-### Checkpoint #3
-* If you don't have it yet, it is a good idea to install [LiveServer](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), especially if you are using VScode.
-![image](https://user-images.githubusercontent.com/69332964/99007366-0fd21f80-2512-11eb-9af9-311d89098c0b.png)
-* To start a local server, click `Go live` at the bottom right of the screen, as shown in the image.
-    * Make sure that you have the entire repo open on VS Code and not just the individual files.
-    * If this is your first time installing LiveServer, you might need to close/quit VS Code and reopen it.
-* Test it out, and see what your HTML page looks like! *It's OK if it's boring, but as I mentioned before, feel free to style it!*
-
-After that, you're done with the frontend. It's time to use JavaScript!
-
-**To move on, commit your code and comment a description of what you completed!**
-
+---
+files: n/a
+stepType: IssueComment
+scripts: testStep1.js
+week: 3
+step: 1
+name: Week 3 Step 1
+---
+
+## Week 3 Step 1
+
+In week 2, you should've learned the basics of how to create an Azure Function and use the Giphy and Twilio APIs. This week, we'll be looking into using [Azure Cosmos DB](https://azure.microsoft.com/en-us/services/cosmos-db/), a powerful NoSQL database.

```

Comment:
Everything looks great, just change this part ^^
![image](https://user-images.githubusercontent.com/69332964/114106882-c4902300-989d-11eb-8f81-af66cfbde06a.png)

One of the APIs will actually be in Week 3 :)